app/assets/javascripts/app/views/searchbar_view.js

```diff
@@ -0,0 +1,70 @@
+// @license magnet:?xt=urn:btih:0b31508aeb0634b347b8270c7bee4d411b5d4109&dn=agpl-3.0.txt AGPL-v3-or-Later
+app.views.SearchBar = app.views.Base.extend({
+  initialize: function(){
+    this.searchForm = this.$el;
```

Comment:
I think you could just use `this.$el` everywhere. `this.searchForm.find(...)` could be written as `this.$(...)`.